redfish/manager.go

```diff
@@ -304,6 +308,7 @@ func (manager *Manager) UnmarshalJSON(b []byte) error {
 		ManagerForSwitches      common.Links
 		ManagerForSwitchesCount int `json:"ManagerForSwitches@odata.count"`
 		ManagerInChassis        common.Link
+		Oem                     interface{}
```

Comment:
To keep with Go style, name these OEM and use `json:"Oem"` to match the JSON name (though IIRC, the json serializer will handle that).

Reply:
I think for OEM properties, we can likely use something better than interface{}. For OEM properties, the specification defines them as having a name as a key and a JSON object as a property. So I think we should at least provide that list of keys with interface{} values so the consumer can decide what to do once they identify the value they want.

Reply:
I like map[string]interface too, but seems like it will have crash in common.(*Entity).Update because it tries to compare map and decide if it need update. I can even change `Update` method if you like, but feel it is more than this PR.
```
panic: runtime error: comparing uncomparable type map[string]interface {} [recovered]
	panic: runtime error: comparing uncomparable type map[string]interface {}

goroutine 18 [running]:
testing.tRunner.func1(0xc000124100)
	/usr/local/go/src/testing/testing.go:830 +0x392
panic(0x1316f60, 0xc0000ad8a0)
	/usr/local/go/src/runtime/panic.go:522 +0x1b5
github.com/stmcginnis/gofish/common.(*Entity).Update(0xc00009ca80, 0x1380de0, 0xc00009d500, 0x199, 0x1380de0, 0xc00009ca80, 0x199, 0xc000149ee8, 0x3, 0x3, ...)
	/Users/qiwa/workspace/golang/src/github.com/stmcginnis/gofish/common/types.go:63 +0x3b8
github.com/stmcginnis/gofish/redfish.(*Manager).Update(0xc00009ca80, 0xc0000b1080, 0xc00009ca80)
```

```
		fieldName := originalEntity.Type().Field(i).Name
		originalValue := originalEntity.Field(i).Interface()
		currentValue := currentEntity.Field(i).Interface()
		if originalValue != currentValue {
			// TODO: Handle JSON name being different than field name
			payload[fieldName] = currentValue
		}
```